cdap-data-fabric/src/main/java/co/cask/cdap/data/view/ViewAdmin.java

```diff
@@ -59,13 +64,17 @@ public boolean createOrUpdate(Id.Stream.View viewId, ViewSpecification spec) thr
       spec = new ViewSpecification(spec.getFormat(), naming.getTableName(viewId));
     }
     explore.enableExploreStream(viewId.getStream(), spec.getTableName(), spec.getFormat());
-    return store.createOrUpdate(viewId, spec);
+    boolean result = store.createOrUpdate(viewId, spec);
```

Comment:
So ``createOrUpdate`` returns ``false`` when the view already exists, and is updated. We want to update system metadata even when the view is updated, right?